### PR TITLE
Standardization and organization of documentation

### PR DESCRIPTION
**Pull request by JohanSingainy-Moutiapin**
### Presentation

In this project, the documentation FAQ/Glossary has been separate of the platform documentation.
It's always the same documentation generator Sphinx. The new theme has been embed, but some elements are different to distinguish it of the platform documentation.

### Testing 

* Run simple project in english :
```
make singlehtml && make server
```

* Run advanced project with multi-langages :
```
make localizedhtml && make server
```

### CODE REVIEW

**Comment by trouba** [comment]
**TEXT**

- [x] Some sentences don't have a final point:

screen shot 2018-07-23 at 12 24 10

screen shot 2018-07-23 at 12 22 59

**Comment by trouba** [comment]
**TEXT**

- [x] Translation could be improved to avoid repetition (same for english with or or) and "google translate" effect :
 "Vous trouverez ici des réponses à vos questions sur les formats de fichiers, les navigateurs pris en charge ou les métadonnées ou simplement une signification de mot"
"Vous trouverez ici des réponses à vos questions sur les formats de fichiers, les navigateurs pris en charge, les métadonnée ou simplement la signification d'un mot."

screen shot 2018-07-23 at 12 25 30

**Comment by trouba** [comment]
**TEXT**

- [x] text
screen shot 2018-07-23 at 13 33 58

**Comment by trouba** [comment]
**BUG**

- [ ] I sometimes loose the language when switching between pages of the main menu
Edit: Checkd if this could come from the "english only" widget page, but even without going on this page I loose the lang
Edit2: The faulty page seems to be "Platform"

**Comment by trouba** [comment]
**BUG**

- [ ] When clicking the anchor icon on titles, it brings me to the title, but its under the header. 
I think the click should just copy the link if this is for sharing.

**Comment by trouba** [comment]
**BUG**

- [x] Content of the FAQ can't be accessed through tabs (deploy all, deploy article)
https://betahelp.opendatasoft.com/faq-glossary/en/

**Comment by trouba** [comment]
**BUG**

- [x] On the API page, this sentence is repeated twice
screen shot 2018-07-23 at 13 51 34

**TEXT**

- [x]  Maybe the first comma of this sentence should be removed
"Our APIs reference, with endpoints, parameters you can use and consoles to run interactive queries"
VS
"Our APIs reference with endpoints, parameters you can use and consoles to run interactive queries."

**Comment by trouba** [comment]
**USABILITY**

- [x] In the widget menu: 
When I click the seconde title "Widget tutorials", the Widget API entries are removed. It's a strange behavior for a vertical menu. Related content should under its title.
Maybe Widget tutorial should be first ?

**Comment by trouba** [comment]
**USABILITY**

- [x] On mobile, the lang selector has the same importance then the main menu:
screen shot 2018-07-23 at 13 56 54

Maybe a separator or a smaller text could help differentiate them

**Comment by trouba** [comment]
**BUG**

- [x] The custom logos used for each page are not aligned so its make a little jump when switching between pages 
screen shot 2018-07-23 at 14 01 09
And it also make the menu jump

**Comment by trouba** [comment]
**BUG**

- [x] The alt text on the header and footer logo reads "OpenDataSoft Documentation" instead of "OpenDataSoft Help hub"

**BUG**

- [x] For other pages, the alt should be "OpenDataSoft blabla documentation", ie "OpenDataSoft APIs documentation"

**Comment by trouba** [comment]
**USABILITY**

- [ ] The page named "Platform" has a "Documentation" logo
screen shot 2018-07-23 at 14 05 40

**Comment by trouba** [comment]
It's strange that "Discovery" doesn't use the same header and is an external link, that should somehow be represented, with the following icon for example, and be separated from the others
image

**Comment by trouba** [comment]
**USABILITY**

- [x] The "expand all" icons are different from icons used in the questions list (+/- vs up down arrow) and the style difference between the two links could indicate that the question links are disabled. Maybe the it should use the same icons and change the "Expand all" style so it doesn't look exactly like the questions links with a different behavior.
screen shot 2018-07-23 at 14 21 19
screen shot 2018-07-23 at 14 21 20

**Comment by trouba** [comment]
**USABILITY**

- [x] In mobile, the menu changes all the time:
On the Help Hub page, there is one "Menu" with the Help hub menu
screen shot 2018-07-23 at 14 45 35

On the "Platform" page, there is one "Menu" with the documentation menu, and one "Help Hub" menu with the hep hub menu
screen shot 2018-07-23 at 14 48 22

On the "FAQ" page, there is one "Help Hub" menu, with a "Help Hub" menu
screen shot 2018-07-23 at 14 49 20

On the "Widgets" page there is one "Menu" with the widget menu, and one "Help hub" menu with the help hub menu
screen shot 2018-07-23 at 14 50 52

On the API page there is one "Menu" with the API menu
screen shot 2018-07-23 at 14 51 35


Maybe the Help hub menu should always be on the same side (the right side), always called "Help hub menu" and the contextual menu should always on the left side, called "Blabla menu", like "APIs menu", "Widgets menu" ...

**Comment by trouba** [comment]
**USABILITY**

- [x] Lang selector links a should have a title with the complete lang name in the target lang

**Comment by trouba** [comment]
**BUG**

- [x] Imgs alt in cards are all the same "OpenDataSoft Documentation"

**Comment by trouba** [comment]
**USABILITY**

- [x] Card content should be aligned top so there is no gap when text are not the same length
screen shot 2018-07-23 at 14 59 05

**Comment by trouba** [comment]
There is no footer on the widget page (so no lang selector)
https://betahelp.opendatasoft.com/widgets/#/api/

**Comment by trouba** [comment]
**USABILITY**

- [x] The lang buttons and menu buttons style are inverted 
screen shot 2018-07-23 at 15 26 35

**Comment by trouba** [comment]
**USABILITY**

- [x] There is nothing explaining what the help hub is, and the help thing make me think first to "Support", and the is only a very small and almost hidden link for that. Maybe this should be a dedicated card as well with a mailto behavior on click
screen shot 2018-07-23 at 15 29 22

**USABILITY**

- [x] And this text is hidden by default on my screen because of the fixed footer

screen shot 2018-07-24 at 10 48 35

**Comment by trouba** [comment]
**BUG**

- [x] In FAQ, the Edge icon alt is "Internet Explorer"
- [x] Both icon are the same (?)

**Comment by trouba** [comment]
**USABILITY**

- [x] The help hub icon and the documentation icon are the same

**Comment by trouba** [comment]
**USABILITY**

- [x] On mobile, the menu scrolling is not very smooth
https://css-tricks.com/snippets/css/momentum-scrolling-on-ios-overflow-elements/
PIng me if you want a demo

**Comment by trouba** [comment]
**USABILITY**

- [ ] not sure a fixed footer at the bottom of the page is necessary on mobile

**Comment by trouba** [comment]
**USABILITY**

- [ ] On mobile, the touch area on menu link is only on the text, it should take the all div

**Comment by trouba** [comment]
**USABILITY**

- [ ] Not sure of this, but clicking outside the menu doesn't close it, it didn't bothered me on desktop but on mobile it tried it a few time without even thinking about it (and of course clicked on an item and got taken to an other page)

**Comment by trouba** [comment]
**BUG**

- [x] The mobile breakpoint is not the same on the widget page
screen shot 2018-07-23 at 16 00 52
screen shot 2018-07-23 at 16 00 56

On Edge, there are more problems with breakpoints and can lead to have both mobile and desktop styles applied:

screen shot 2018-07-24 at 10 20 10

**Comment by trouba** [comment]
**USABILITY**

- [ ] Since there is a graphic identity for each section, why don't we use this for the hub ?
screen shot 2018-07-23 at 16 12 08
Sometimes we use the same logo in the card and in the header (Widgets), sometimes an approaching version (FAQ), sometimes something different (Paltform). We could enforce pages/section identity by always using the same logo
screen shot 2018-07-24 at 10 59 21

**Comment by trouba** [comment]
**USABILITY**

- [x] I don't understand the navigation of this site. It is the help hub, so it's a unique entry point, and the root is the home (except for Discovery, that is an external site). 
The logo changes on each page, so I can't go back to the home with the standard behavior (clicking on the main logo) and what is called the help hub, is in fact not the hub but just a "menu" page with links to others pages. 
Having a link on the logo is useless since it stays on the same page. It would be better to just remove that link since it shows an action that has no consequence.

**Comment by trouba** [comment]
**USABILITY**

- [ ] Why not use the Discovery logo on the Discovery card ? The magnifying glass icon is misleading and is usually associated with search features.

**Comment by trouba** [comment]
**BUG**

- [x] On the API page, clicking the logo goes back to the Help Hub page, this is not consistent with the behavior on the other pages (but at least it's a standard behavior).

**Comment by trouba** [comment]
**BUG**

- [x] FAQ title is not translated in NL
screen shot 2018-07-24 at 10 44 22

File: scripts/helpscout.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
+from __future__ import print_function
```
Comment (vbarbaresi) [inline scripts/helpscout.py]: I don't see the point of this kind of script being open source in this repo

**Comment by vbarbaresi** [comment]
It's a shame that all the .png resources are on ods-documentation and duplicated here

**Comment by JohanSingainy-Moutiapin** [comment]
Thanks @vbarbaresi

**Comment by trouba** [comment]
**BUG**

- [ ] In FAQ, screenshot are not localized
screen shot 2018-07-25 at 10 51 58

**Comment by trouba** [comment]
**BUG**
- [ ] In the Glossary, some sources links are dead, like all whitehouse.gov links, and some others

**Comment by trouba** [comment]
**USABILITY**

- [x] When "Expand all" is clicked and a question is closed, the "expand all" button changes to "collapse all".

**Pull request merged**